Source/GitReleaseManager.Cli/GitReleaseManager.Tool.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageProjectUrl>https://github.com/GitTools/GitReleaseManager</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>github release notes create export</PackageTags>
-    <Description>Tool for creating and exporting releases for software applications hosted on GitHub.</Description>
+    <Description>Tool for creating and exporting releases for software applications.</Description>

```

Comment:
instead of removing `hosted on GitHub`, maybe change it to `hosted on a remote provider`?

Reply:
Done